packages/esm-service-queues-app/src/queue-table/cells/queue-table-action-cell.component.tsx

```diff
@@ -9,6 +9,31 @@ export function QueueTableActionCell({ queueEntry }: QueueTableCellComponentProp
 
   return (
     <div>
+      {queueEntry.previousQueueEntry == null ? (
+        <Button
+          kind="ghost"
+          aria-label="Actions"
+          onClick={() => {
+            const dispose = showModal('void-queue-entry-modal', {
+              closeModal: () => dispose(),
+              queueEntry,
+            });
+          }}>
+          {t('void', 'Void')}
+        </Button>
+      ) : (
+        <Button
+          kind="ghost"
+          aria-label="Actions"
+          onClick={() => {
+            const dispose = showModal('undo-transition-queue-entry-modal', {
+              closeModal: () => dispose(),
+              queueEntry,
+            });
+          }}>
+          {t('moveBack', 'Move back')}
```

Comment:
This isn't really correct.  "Move back" to me, implies that the original transition was correct, and now they are done there and need to go back in the other queue again.  This is really here as an _alternative_ to "Delete", which is essentially "Delete and reactivate previous queue entry" or "Undo this transition".  We need to make it explicit that this is here to _revert_ the previous action, as if it had never happened.